ynput/ayon-core
# Publisher: Screen capture fixes

## Changelog Description
Disable multi-screen screengrab to avoid issues related to scale ratio.

## Additional info
Now each screen has own dialog window (which makes it faster BTW). With that we make sure that each screen is covered with the correct size.

Removed option to capture screen across multiple screens. That is related to issues connected to different scale ration causing output image size issues.

## NOTE
Surprise for me, I don't know how to take screenshot of desktop, it only take screenshot of background and windows of the process it is running in.

## Testing notes:
Some of the issues happened everywhere, but on macOs the biggest issue was that capture area was randomly moved on scaled multi-screen, so it was not possible to capture screen of some areas at all.
- ideally at least of your screens should have some scaling
1. Open publisher in host of your choice (ideally traypublisher and one DCC with Qt like Nuke or Maya).
2. Create any instance.
3. Select the instance and go to Publish tab.
4. Hover over thumbnail widget and press "Take screenshot".
5. It should be possible to take screenshot on any screen.
6. The output should match the real selection.
